test/blackbox/utils/Utils.ts

```diff
@@ -0,0 +1,67 @@
+import {promises as fs} from 'fs';
+import * as path from 'path';
+import { AbstractGenerator, FormatHelpers, OutputModel } from '../../../src';
+import {promisify} from 'util';
+import {exec} from 'child_process';
+const promiseExec = promisify(exec);
+
+/**
+ * Exec a command and if any errors occur reject the promise.
+ * 
+ * @param absolutePathToFile 
+ * @param generator  
+ */
+ export async function generateModels(absolutePathToFile: string, generator: AbstractGenerator) {
+  const inputFileContent = await fs.readFile(absolutePathToFile);
+  const input = JSON.parse(String(inputFileContent));
+  return await generator.generate(input);
+}
+/**
+ * Execute a command and if any errors occur reject the promise.
+ * 
+ * @param command 
+ */
+ export async function execCommand(command: string) {
+  try {
+    const { stderr } = await promiseExec(command);
+    if(stderr !== '') {
+      Promise.reject(stderr);
+    }
+  } catch (e) {
+    Promise.reject(e);
+  }
+}
+
+/**
+ * Render all models to separate files in the same directory
+ * 
+ * @param generatedModels to write to file
+ * @param outputPath absolute path to output directory
+ */
+export async function renderModelsToSeparateFiles(generatedModels: OutputModel[], outputPath: string) {
+  await fs.rm(outputPath, { recursive: true, force: true });
+  await fs.mkdir(outputPath, { recursive: true });
+  for(const outputModel of generatedModels) {
+    const outputFilePath = path.resolve(outputPath, `${FormatHelpers.toPascalCase(outputModel.model.$id || 'undefined')}.java`);
+    await fs.writeFile(outputFilePath, outputModel.result);
+  }
+}
+
+
+
+/**
+ * Render all models to a single file
+ * 
+ * @param models to write to file
+ * @param outputPath path to output
+ */
+export async function renderModels(generatedModels: OutputModel[], outputPath: string) {
+  const outputDir = path.resolve(__dirname, path.dirname(outputPath));
+  await fs.rm(outputDir, { recursive: true, force: true });
+  await fs.mkdir(outputDir, { recursive: true });
+  const output = generatedModels.map((model) => {
+    return model.result;
+  });
+  const outputFilePath = path.resolve(__dirname, outputPath);
+  await fs.writeFile(outputFilePath, output.join('\n'));
```

Comment:
If you have async function which returns some result from promise, then last `await` is unnecessary, because finally you return Promise. It's only a tip, `return await` is not a problem for me :)

Reply:
Gonna leave it for this function, more clean to look at when it returns void.